tensorboard/plugins/graph/tf_graph_board/tf-graph-board.ts

```diff
@@ -24,6 +24,18 @@ import {LegacyElementMixin} from '../../../components/polymer/legacy_element_mix
 import {ColorBy} from '../tf_graph_common/view_types';
 import {Hierarchy} from '../tf_graph_common/hierarchy';
 
+/**
+ * Some UX features, such as 'color by structure', rely on the 'template'
+ * information of a graph. This can be very expensive to compute when the
+ * graph is too large. This object's constants determine what constitutes
+ * a 'large' graph. Graphs that exceed all these constraints should not
+ * have templates computed by default.
+ */
+const MAX_GRAPH_SIZE_TO_COMPUTE_TEMPLATES = {
+  maxNodeCount: 10000,
+  maxEdgeCount: 10000,
+};
```

Comment:
style thing: often, consts invert the casing :P 

i.e.,
```ts
const MaxGraphSizeToComputeTemplates = {
  MAX_NODE_COUNT: 10000,
  MAX_Edge_COUNT: 10000,
};

```

Reply:
Done.  I'll remember for next time!